packages/spatial/src/physics/classes/Physics.test.tsx

```diff
@@ -1215,7 +1216,24 @@ describe('Physics : Rapier->ECS API', () => {
         removeEntity(testEntity)
         return destroyEngine()
       })
-
+      it('should not do anything for an entity that has a NetworkObjectComponent but no NetworkAuthorityComponent', () => {
```

Comment:
Let's be clearer with the language here, rather than "should not do anything" let's say "should not update pose and velocity"